data/json/npcs/TALK_ALLY_TUTORIAL.json

```diff
@@ -0,0 +1,200 @@
+    "responses": [
+      { "text": "Forget I asked.", "topic": "TALK_FRIEND" },
+      { "text": "Skip it, let's get going.", "topic": "TALK_DONE" },
+      { "text": "What's that about giving instructions?", "topic": "TALK_ALLY_TUTORIAL_TALK" },
+      { "text": "We can talk with radios?", "topic": "TALK_ALLY_TUTORIAL_RADIO" },
+      { "text": "How do shouted commands work?", "topic": "TALK_ALLY_TUTORIAL_SHOUTS" },
+      { "text": "How does fighting work?", "topic": "TALK_ALLY_TUTORIAL_COMBAT" },
+      { "text": "What's that about training?", "topic": "TALK_ALLY_TUTORIAL_TRAINING" },
+      { "text": "Tell me about you carrying stuff", "topic": "TALK_ALLY_TUTORIAL_MULE" },
+      { "text": "You can perform first aid?", "topic": "TALK_ALLY_TUTORIAL_MEDIC" },
+      { "text": "Tell me about faction camps.", "topic": "TALK_CAMP_TUTORIAL_1" },
+      { "text": "What do you mean, you can go places?", "topic": "TALK_ALLY_TUTORIAL_MOVEMENT" },
+      { "text": "What do you do as a guard?", "topic": "TALK_ALLY_TUTORIAL_GUARD" },
+      { "text": "What about chit-chatting and your background?", "topic": "TALK_ALLY_TUTORIAL_SOCIAL" },
+      { "text": "Anything else I should know?", "topic": "TALK_ALLY_TUTORIAL_MISC" }
+    ]
+  },
+  {
+    "id": "TALK_ALLY_TUTORIAL_TALK",
+    "type": "talk_topic",
+    "dynamic_line": "If we're next to each other, you can just bump into me and we'll start talking, right?  But if I'm farther away, you're going to have to shout a bit (use the 'C'hat command) for me to hear you.  You'll need to see me for us to have a conversation.  Or we can talk by radios if we both have them. \nWhen we talk, you can give me instructions about how to fight or when to sleep or whatever.  I'll mostly do them, and you can ask me what my current instructions are.  Sometimes you'll give me two sets of instructions: a normal set, and an override for a specific situation.  I'll tell you which instructions are overriden.  You can set and clear overrides with shouted commands.",
+    "responses": [
+      { "text": "We can talk with radios?", "topic": "TALK_ALLY_TUTORIAL_RADIO" },
+      { "text": "How do shouted commands work?", "topic": "TALK_ALLY_TUTORIAL_SHOUTS" },
+      {
+        "text": "Thanks for the explanation.  I wanted to know more about something else.",
+        "topic": "TALK_ALLY_TUTORIAL"
+      },
+      { "text": "Thanks.  I have some things for you to do.", "topic": "TALK_FRIEND" },
+      { "text": "Skip it, let's get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ALLY_TUTORIAL_RADIO",
+    "type": "talk_topic",
+    "dynamic_line": {
+      "and": [
+        "is_by_radio": "Are you making a joke?  We're using two-way radios to talk right now.  Whatever, I'll go over it again in case you forgot.\n",
```

Comment:
This doesn't seem to be valid json, array with a key/value pair?
Should be ``{ "is_by_radio": "Are you making a joke?  We're using two-way radios to talk right now.  Whatever, I'll go over it again in case you forgot.\n" },`` ?

Reply:
I have a 100% failure rate on "this is an obvious and trivial change, I don't need to test it" so you'd think I'd learn.